roles/kubernetes_monitoring/defaults/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+##########################
+# kubernetes_monitoring
+
+kubernetes_monitoring_chart_source: "dnationcloud/dnation-kubernetes-monitoring-stack"
+kubernetes_monitoring_chart_version: "3.5.0"
+
+## work directory inside the osism-ansible container:
+kubernetes_monitoring_work_directory: /tmp/kubernetes_monitoring/configuration
+## template directory containing `kubernetes_monitoring.yml`, e.g. OSISM configuration directory
+kubernetes_monitoring_template_directory: .
+## kubernetes namespace in which the operator should be installed
+kubernetes_monitoring_namespace: kubernetes-monitoring
+
+##########################
+# openstack_exporter
+
+enable_openstack_exporter: false
+openstack_exporter_chart_source: "oci://registry.scs.community/openstack-exporter/prometheus-openstack-exporter"
+openstack_exporter_chart_version: "0.4.5"
```

Comment:
Add a renovate line before this line. Probably `# renovate: datasource=docker depName=registry.scs.community/openstack-exporter/prometheus-openstack-exporter`

Reply:
see 15b597fe13a1c91db5b87a72f90c44fe681698b2
We can take a GitHub release as a version source also in this case